Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -64,47 +68,56 @@ public class CustomerIOImplementation: CustomerIOInstance {
     public func identify<RequestBody: Encodable>(
         identifier: String,
         body: RequestBody,
-        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void,
         jsonEncoder: JSONEncoder? = nil
     ) {
-        identifyRepository
-            .addOrUpdateCustomer(identifier: identifier, body: body, jsonEncoder: jsonEncoder) { [weak self] result in
-                DispatchQueue.main.async { [weak self] in
-                    guard self != nil else { return }
-
-                    switch result {
-                    case .success:
-                        return onComplete(Result.success(()))
-                    case .failure(let error):
-                        return onComplete(Result.failure(error))
-                    }
-                }
-            }
+        if let currentlyIdentifiedProfileIdentifier = profileStore.identifier {
+            // TODO: add to background queue delete device token from currently registered profile.
```

Comment:
why?

Reply:
Because if person A is no longer identified and person B is, we don't want push notifications to be sent to the device that are only relevant to person A while person B is now logged into the app. 

Privacy + relevance. The customer will be able to configure this automatic behavior.

Reply:
gotcha. this reads strangely since it's not also showing the check that the identifier has changed, but yes that behaviour is what we want in the case where an identify is called and the identifier has changed

Reply:
I don't think we should check if the identifier has changed because what if the customer calls this function multiple times simply to add some more user attributes?